src/frontend/src/binder/mod.rs

```diff
@@ -77,24 +80,40 @@ pub struct Binder {
     cte_to_relation: HashMap<String, (BoundQuery, TableAlias)>,
 
     search_path: SearchPath,
+
+    in_create_mv: bool,
 }
 
 impl Binder {
-    pub fn new(session: &SessionImpl) -> Binder {
+    fn new_inner(session: &SessionImpl, in_create_mv: bool) -> Binder {
+        let now_nsecs = Local::now().timestamp_nanos();
```

Comment:
To keep the result of batch queries consistent with streaming, it's better to use the timestamp of the current epoch. (had mentioned in https://github.com/risingwavelabs/risingwave/issues/7139#issuecomment-1367734075)

Reply:
> se the timestamp of the current epoch

sry. fixed.